airbyte-webapp/src/locales/en.json

```diff
@@ -436,6 +436,9 @@
   "connector.releaseStage.beta": "beta",
   "connector.releaseStage.custom": "custom",
   "connector.releaseStage.generally_available": "generally available",
+  "connector.acknowledgeTerms": "I acknowledge these terms",
+  "connector.termsModal.text": "<b>Alpha</b> connectors are in active development. The release may not be feature complete and breaking changes may be introduced. \n\nOn occasion, we will reach out to users to gather feedbacks so that we can improve the connector.",
+  "connector.connectorsInDevelopment": "<b>Alpha connectors</b> are in active development. The release may not be feature complete and breaking changes may be introduced.\nOn occasion, we will reach out to users to gather feedbacks so that we can improve the connector.",
```

Comment:
```suggestion
  "connector.connectorsInDevelopment": "<b>Alpha connectors</b> are in active development. The release may not be feature complete and breaking changes may be introduced. On occasion, we will reach out to users to gather feedback so that we can improve the connector.",
```